velox/serializers/PrestoSerializer.cpp

```diff
@@ -449,6 +450,20 @@ void readColumns(
     std::vector<VectorPtr>* result,
     bool useLosslessTimestamp);
 
+/// Read RLE encoded vector
+void readConstantVector(
```

Comment:
Consider returning vector instead of using an output parameter.

```
return BaseVector::wrapInConstant(size, 0, children[0]);
```

Reply:
All the readBlaBlaVector methods in this class use the output parameter, I made it consistent with the rest of methods. I also planned to look into reusability of the constant and the wrapped vector a bit later.

Are you sure it would be better to return it instead of using the output parameter?